### PR TITLE
ODC-7292: Add new quickstart for installing Cryostat

### PR DESCRIPTION
**Pull request by lokanandaprabhu**
New quickstart for installing Cryostat. This was created by the Cryostat team, and reviewed by their docs team. This addresses on of the 4.14 epic - https://issues.redhat.com/browse/ODC-7111

### CODE REVIEW

**Comment by lokanandaprabhu** [comment]
Propage labels from the epic:

/label docs-approved
/label px-approved

File: quickstarts/consolequickstart-install-cryostat.yaml [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  introduction: >-
+    Cryostat is a container-native Java application based on JDK Flight Recorder (JFR). You can use it to monitor the performance of the Java Virtual Machine (JVM) for containerized workloads that run on an OpenShift cluster.
+
+    1. You can deploy Cryostat in a container in an OpenShift project that hosts your containerized Java applications. 
+
+    1. You can create JVM targets that correspond to the JVM instances that you use to run your containerized workload. 
+
+    1. You can connect Cryostat to the JVM targets to record and analyze data about heap and non-heap memory usage, thread count, garbage collection, and other performance metrics for each JVM target.
+
+    To start using Cryostat, install the cert-manager Operator for Red Hat OpenShift and Red Hat build of Cryostat Operators on your cluster.
```
Comment (jerolimov) [inline quickstarts/consolequickstart-install-cryostat.yaml:33]: This line isn't shown as separate paragraph:

image

One extra line above this sentence fix this:

```suggestion

        To start using Cryostat, install the cert-manager Operator for Red Hat OpenShift and Red Hat build of Cryostat Operators on your cluster.
```

File: quickstarts/consolequickstart-install-cryostat.yaml [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+          From the **Administrator** perspective, go to **Installed Operators** from the [Operators]{{highlight qs-nav-operators}} section of the navigation. 
+
+          - Go to the Red Hat build of Cryostat operator and click the **Cryostat** tab.
+
+          - Do you see the Cryostat instance in the Cryostats list?
+
+          - Click the Cryostat instance in the **Cryostats** list. 
+
+            - You should see a URL to the Cryostat application in Application URL.
+            - In the **Cryostat Conditions** table, do you see the **TLSSetupComplete** and **MainDeploymentAvailable** types set to `True`?
```
Comment (jerolimov) [inline quickstarts/consolequickstart-install-cryostat.yaml:141]: These two sub-list items aren't shown as sublist. Is it possible:

image

You need two more spaces per item:

```suggestion
              - You should see a URL to the Cryostat application in Application URL.
              - In the **Cryostat Conditions** table, do you see the **TLSSetupComplete** and **MainDeploymentAvailable** types set to `True`?
```

**Review by jerolimov — CHANGES_REQUESTED** [review]
Looks almost good, found two tiny layout issues that needs one extra line and four spaces :smile:

**Review by jerolimov — APPROVED** [review]
Applied the QuickStart on my cluster and it looks good :+1: 

/lgtm
/approve

**Review by jhadvig — APPROVED** [review]
/lgtm
/approve

File: quickstarts/consolequickstart-install-cryostat.yaml [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+            - The cert-manager Operator for Red Hat OpenShift should be shown in the list of Operators
+            - In the **Status** column, do you see the status of **cert-manager Operator for Red Hat OpenShift** as **Succeeded**?
+
+    - title: Install the Red Hat build of Cryostat Operator
+      description: >
+        To install the Red Hat build of Cryostat operator:
+       
+        1. From the **Administrator** perspective, go to the **OperatorHub** from the [Operators]{{highlight qs-nav-operators}} section of the navigation. 
+        
+        1. From the **Project** list, select the OpenShift project that hosts your containerized Java applications.
```
Comment (ebaron) [inline quickstarts/consolequickstart-install-cryostat.yaml:76]: Sorry, this point should be in the "Create a Cryostat instance" page instead. The `View installed Operators in Namespace` step changes the project back to `openshift-operators`.

```suggestion
```

File: quickstarts/consolequickstart-install-cryostat.yaml [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+
+          - The Red Hat build of Cryostat should be shown in the list of Operators.
+
+          - In the **Status** column, do you see the status of the Red Hat build of Cryostat as `Succeeded`?
+    - title: Create a Cryostat instance
+      description: >
+        To create a Cryostat instance:
+
+        1. From the **Administrator** perspective, go to **Installed Operators** from the [Operators]{{highlight qs-nav-operators}} section of the navigation. 
+
```
Comment (ebaron) [inline quickstarts/consolequickstart-install-cryostat.yaml:110]: Moved from the previous page:
```suggestion
        
        1. From the **Project** list, select the OpenShift project that hosts your containerized Java applications.
        
```

**Review by ebaron — COMMENTED** [review]
Sorry, noticed one last problem when working through the Quickstart.

**Comment by jerolimov** [comment]
Adding QE approved since this change only adds new content. @sanketpathak FYI. Please let me/us know if you have any issue with this.

/label qe-approved

File: quickstarts/consolequickstart-install-cryostat.yaml (hunk 3 above)
Comment (lokanandaprabhu) [reply quickstarts/consolequickstart-install-cryostat.yaml:76]: Created PR - https://github.com/openshift/console-operator/pull/773 to handle this. PTAL.

File: quickstarts/consolequickstart-install-cryostat.yaml (hunk 4 above)
Comment (lokanandaprabhu) [reply quickstarts/consolequickstart-install-cryostat.yaml:110]: Created PR - https://github.com/openshift/console-operator/pull/773 to handle this. PTAL.

**Pull request merged**